ui/common/HolocatQuestion.tsx

```diff
@@ -26,11 +26,11 @@ const HolocatQuestion = ({
         id={id}
         theme={theme || ''}
         content={question}
-        parentClassName="inline-flex align-bottom min-h-fit min-w-fit"
+        parentClassName="inline-flex align-middle min-h-fit min-w-fit mx-1"
```

Comment:
I think actually `align-sub` is the class we might want to use. It seems more inline than `align-middle`

Reply:
```suggestion
        parentClassName="inline-flex align-sub min-h-fit min-w-fit mx-1"
```